python/mxnet/ndarray/numpy/_op.py

```diff
@@ -882,3 +906,162 @@ def sqrt(x, out=None, **kwargs):
     This function only supports input type of float.
     """
     return _unary_func_helper(x, _npi.sqrt, _np.sqrt, out=out, **kwargs)
+
+
+@set_module('mxnet.ndarray.numpy')
+def floor(x, out=None, **kwargs):
+    r"""
+    Return the floor of the input, element-wise.
+    The floor of the scalar `x` is the largest integer `i`, such that
+    `i <= x`.  It is often denoted as :math:`\lfloor x \rfloor`.
+
+    Parameters
+    ----------
+    x : ndarray or Python int
```

Comment:
Are you sure this accepts 'Python int' only except ndarrays? Does `floor(2.5)` not work?

Reply:
fixed